packages/components/src/components/accordion/accordion.stories.ts

```diff
@@ -90,87 +92,15 @@ export const SummaryLength = {
-  },
-  render: (args: any) => {
-    return html`
-      ${['default', 'summary', 'expand-icon', 'collapse-icon'].map(slot =>
-        generateTemplate({
-          axis: {
-            x: {
-              type: 'slot',
-              name: slot,
-              title: 'slot=...',
-              values: [
-                {
-                  value:
-                    slot === 'default'
-                      ? `<div class="slot slot--border slot--background slot--text h-16">Default slot</div>`
-                      : `<div slot='${slot}' class="${cx(
-                          'slot slot--border slot--background h-6',
-                          slot === 'summary' ? 'w-[100%]' : 'w-6'
-                        )}"></div>`,
-                  title: slot
-                }
-              ]
-            }
-          },
-          constants: [
-            { type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' },
-            { type: 'attribute', name: 'open', value: slot === 'collapse-icon' || slot === 'default' ? true : false }
-          ],
-          args: overrideArgs({ type: 'slot', name: 'default', value: '' }, args)
-        })
-      )}
-    `;
-  }
-};
-
-export const Parts = {
-  parameters: {
-    controls: { exclude: ['base', 'header', 'summary', 'summary-icon', 'summary-border', 'content', 'content__slot'] }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'template',
-          name: 'sd-accordion::part(...){outline: solid 2px red}',
-          values: ['base', 'header', 'summary', 'summary-icon', 'summary-border', 'content', 'content__slot'].map(
-            part => {
-              const outlineOffset = part === 'summary-border' ? '' : 'outline-offset: -2px';
-              return {
-                title: part,
-                // Added an outline-offset to make the outline visible for content__slot
-                value: `<style>#part-${part} sd-accordion::part(${part}){outline: solid 2px red; ${outlineOffset};}</style><div id="part-${part}">%TEMPLATE%</div>`
-              };
-            }
-          )
-        }
-      },
-      constants: [
-        { type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' },
-        { type: 'attribute', name: 'open', value: true }
-      ],
-      args
-    });
-  }
-};
-
-/**
- * sd-accordions are fully accessibile via keyboard.
- */
-
-export const Mouseless = {
-  render: (args: any) => {
-    return html`<div class="mouseless">${generateTemplate({ args })}</div>`;
-  },
-
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-accordion');
-    await waitUntil(() => el?.shadowRoot?.querySelector('header'));
-    await userEvent.type(el!.shadowRoot!.querySelector('header')!, '{space}', { pointerEventsCheck: 0 });
-  }
+export const SlottedIcons = {
```

Comment:
Those slots are meant for theming purposes. E. g. https://www.attrax.lu will use/uses our components as well but they shouldn't the official icons. THerefore I think we should remove this here and make this less "explicit" to not invite people to do this. What do you think @coraliefeil?

Reply:
These have been removed.